transitions/multi_form_alerts.js

```diff
@@ -125,33 +132,33 @@ const getPhonesWithDuplicates = (recipients, countedReports) => {
   return getPhonesOneRecipient(recipients, countedReports);
 };
 
-const validateConfig = (alert) => {
-  if (!alert.isReportCounted ||
-      !alert.numReportsThreshold ||
-      !alert.message ||
-      !alert.recipients ||
-      !alert.timeWindowInDays) {
-    throw new Error(`Bad config for multi_form_alerts. Expecting fields isReportCounted, ` +
-      `numReportsThreshold, message, recipients, timeWindowInDays. Found ${JSON.stringify(alert)}`);
-  }
-  alert.timeWindowInDays = parseInt(alert.timeWindowInDays);
-  if (isNaN(alert.timeWindowInDays)) {
-    throw new Error('Bad config for multi_form_alerts. Expecting "timeWindowInDays" to be an integer. ' +
-      'E.g "timeWindowInDays": "3"');
-  }
-  alert.numReportsThreshold = parseInt(alert.numReportsThreshold);
-  if (isNaN(alert.numReportsThreshold)) {
-    throw new Error('Bad config for multi_form_alerts. Expecting "numReportsThreshold" to be an integer. ' +
-      'E.g "numReportsThreshold": "3"');
-  }
-  if(!_.isArray(alert.recipients)) {
-    throw new Error('Bad config for multi_form_alerts. Expecting "recipients" to be an array of strings. ' +
-      'E.g "recipients": ["+9779841452277", "countedReports[0].contact.phone"]');
-  }
-  if (alert.forms && (!_.isArray(alert.forms))) {
-    alert.forms = null;
-    logger.warn('Bad config for multi_form_alerts. Expecting "forms" to be an array of form codes. Continuing without "forms", since it\'s optional.');
-  }
+const validateConfig = () => {
+  const alertConfig = getAlertConfig();
+  alertConfig.forEach(alert => {
+    requiredFields.forEach(field => {
+      if (!alert[field]) {
+        throw new Error(`Bad config for multi_form_alerts. Expecting fields: ${requiredFields.join(', ')}`);
```

Comment:
This is really good. I think we can improve it by being more explicit in what is going wrong, to aid tech leads in fixing the problem. We could add the specific config that failed, for example. I think the best approach would be to refactor this to run through all configs and all fields, and then output the entire missing set.

From personal experience this is really helpful. It's always frustrating to fix one issue, run it through and then discover the next issue, rinse repeat etc.